draft-ietf-quic-applicability.md

```diff
@@ -202,10 +213,19 @@ idle periods than for TCP. According to a 2010 study ({{Hatonen10}}), UDP
 applications can assume that any NAT binding or other state entry will be
 expired after just thirty seconds of inactivity.
 
+By using a Connection ID, QUIC is designed to be robust to NAT address
+rebinding after a timeout. However, some QUIC connections may not be robust to
+rebinding because the routing infrastructure (in particular, load balancers)
+uses the address/port fourtuple to direct traffic. Furthermore, middleboxes
+with functions other than address translation may still affect the path. In
+particular, firewalls will often not admit server traffic for which it has not
+kept state for corresponding packets from the client.
+
 A QUIC application has three strategies to deal with this issue:
 
 - Ignore it, if the application-layer protocol consists only of interactions
-  with no or very short idle periods.
+  with no or very short idle periods, or the protocol's resistance to NAT
```

Comment:
```suggestion
  with no or very short idle periods, or the application's resistance 
  to NAT
```

Reply:
Not sure I understand this point. Isn't "simply reconnect" on application layer point three below?

Reply:
what this bullet means is the QUIC NAT rebinding mechanism is working fine -- there is no 4-tuple router breaking everything. So this suggested change breaks the meaning, though perhaps it's a better or equally relevant point.